lib/msal-common/src/authority/AuthorityOptions.ts

```diff
@@ -12,4 +12,22 @@ export type AuthorityOptions = {
     cloudDiscoveryMetadata: string;
     authorityMetadata: string;
     azureRegionConfiguration?: AzureRegionConfiguration;
+    azureCloudInstance?: AzureCloudInstance;
 };
+
+export enum AzureCloudInstance {
+    // AzureCloudInstance is not specified.
+    None,
+
+    // Microsoft Azure public cloud. Maps to https://login.microsoftonline.com
+    AzurePublic,
```

Comment:
Can these be given the value of the domain, so that the `getAzureCloundInstanceUrl` method isn't needed?
```suggestion
    AzurePublic = "https://login.microsoftonline.com",
```

Reply:
Not a bad idea. Will do.

Reply:
Actually no, this will make it a "String" enum and that may change the MSAL consistency definition.

Reply:
Hmm okay I see this is how dotnet implements it. In practice, in the enum key is for the customer/application, the value is for us, but we can keep this.

Reply:
So does iOS. Hence keeping this for consistency.